maps/DragonsDen1F.asm

```diff
@@ -1,4 +1,5 @@
 DragonsDen1F_MapScripts:
+; BUG: Clair can give TM24 Dragonbreath twice (see docs/bugs_and_glitches.md)
```

Comment:
This *is* where I decided to put a convenient fix, but it's not related to how the bug occurs. I'd put this line at the `def_scene_scripts` in `DragonsDenB1F_MapScripts`, and add removing it to the fix diff.

Reply:
I moved this to right above `def_callbacks` which is the line immediately proceeding the changed lines.